app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ReservationsController < ApplicationController
+  def new
+    #@reservation = Reservation.new
+    #@cart.data.each do |item_id, quantity|
+      #item = Item.unscoped.find(item_id.to_i)
+      #item.add_quantity(quantity)
+      #@reservation.items << item
```

Comment:
Incorrect indentation detected (column 6 instead of 4).<br>Missing space after #.